src/styles/highlighted-code.css

```diff
@@ -110,3 +110,11 @@
     }
   }
 }
+
+.diff.add {
```

Comment:
I'd like to:
- move it up to the `@layer components`
- adjust scss classes
- add before items as `+` and `-`
- update coloes for dark theme support

```
.has-diff code .line {
  @apply pl-6;

  &.diff {
    &.add {
      @apply bg-code-green-1/10 before:absolute before:left-2 before:text-code-green-1 before:content-['+'];
    }
    &.remove {
      @apply bg-code-red-1/10 before:absolute before:left-2 before:text-code-red-1/50 before:content-['-'];
    }
  }
}
```

![image](https://github.com/neondatabase/website/assets/22715126/a27a97e7-10f2-40bc-be6c-da71107251e1)

![image](https://github.com/neondatabase/website/assets/22715126/626db3aa-0414-4a7d-93fa-758088cfbe0d)

Reply:
Done by yourself.